### ISSUE TITLE
get('8.0') yields null

### ISSUE THREAD

**Issue by filmaj** [dvoiss/android-versions#6]
I am using android-versions 1.2.0.

I would assume that should return api level 26.

`get('26')` works fine.

```
> a.get('7.0')
{ api: 24,
  ndk: 8,
  semver: '7.0',
  name: 'Nougat',
  versionCode: 'N' }
> a.get('26')
{ api: 26,
  ndk: 8,
  semver: '8.0.0',
  name: 'Oreo',
  versionCode: 'O' }
> a.get('8.0.0')
null
> a.get('8.0')
null
```

**Comment by dvoiss** [comment]
Fixed, published to version 1.2.1!

Thanks for the ticket. 👍

**Comment by filmaj** [comment]
Thank you!
